cms/envs/aws.py

```diff
@@ -177,6 +177,7 @@
 
 # Theme overrides
 THEME_NAME = ENV_TOKENS.get('THEME_NAME', None)
+COMP_THEME_DIR = path(ENV_TOKENS.get('COMP_THEME_DIR', COMP_THEME_DIR))
```

Comment:
Nit: I don't like the name ``COMP_THEME_DIR``. It seems unnecessarily confusing and won't be clear what it means without documentation.